### ISSUE TITLE
additional-decorators Document misspelling

### ISSUE THREAD

**Issue by stevezhou6** [MichalLytek/typegraphql-prisma#260]
**Describe the Bug**
It shouldn't be `applyOutputTypeEnhanceMap`, it's `applyOutputTypesEnhanceMap`. 

In case of other output types like AggregateFooBar, you can use the same pattern but this time using the applyOutputTypeEnhanceMap function and OutputTypeConfig or OutputTypesEnhanceMap types:

```ts
const aggregateClientConfig: OutputTypeConfig<"AggregateClient"> = {
  fields: {
    avg: [Extensions({ complexity: 10 })],
  },
};

applyOutputTypeEnhanceMap({
  // separate config
  AggregateClient: aggregateClientConfig,
  // or an inline one
  ClientAvgAggregate: {
    fields: {
      _all: [Extensions({ complexity: 10 })],
      age: [Authorized()],
    },
  },
});
```

**Comment by MichalLytek** [comment]
Please create a PR with the fix 😉

**Comment by MichalLytek** [comment]
Fixed by #339 🔒
